buzsaki_lab_to_nwb/yuta_code_visual_cortex/metadata.yml

```diff
@@ -0,0 +1,28 @@
+NWBFile:
+  related_publications:
+    Senzai, Y., Fernandez-Ruiz, A., & Buzsáki, G. (2019). Layer-specific physiological features and interlaminar
+    interactions in the primary visual cortex of the mouse. Neuron, 101(3), 500-513.
+  experiment_description:
+    The relationship between mesoscopic local field potentials (LFPs) and single-neuron firing in the multi-layered 
+    neocortex is poorly understood. Simultaneous recordings from all layers in the primary visual cortex (V1) of the 
+    behaving mouse revealed functionally defined layers in V1. The depth of maximum spike power and sink-source 
+    distributions of LFPs provided consistent laminar landmarks across animals. Coherence of gamma oscillations 
+    (30–100 Hz) and spike-LFP coupling identified six physiological layers and further sublayers. Firing rates, 
+    burstiness, and other electrophysiological features of neurons displayed unique layer and brain state dependence.
+    Spike transmission strength from layer 2/3 cells to layer 5 pyramidal cells and interneurons was stronger during
+    waking compared with non-REM sleep but stronger during non-REM sleep among deep-layer excitatory neurons. A subset 
+    of deep-layer neurons was active exclusively in the DOWN state of non-REM sleep. These results bridge mesoscopic 
+    LFPs and single-neuron interactions with laminar structure in V1.
+  institution: NYU
+  lab: Buzsaki
+Subject:
+  species: Mus musculus
+  sex: M
+  weight: 28-35g
+  age: 3-8 months
+Ecephys:
+  Device:
+    - name: Device_ecephys
```

Comment:
@h-mayorquin This line is actually quite important in the metadata structure; if you don't include the name, the `deep_dict_update` won't properly merge information, and if you rename it to something other than the default of `Device_ecephys`, you would also have to update linkages from other objects that reference the device.

Reply:
Right, I think we had this in the file of the convert class, correct?. I did not know if everything should be in the metadata.yaml file. I guess it makes sense to push as much as possible into it. 

Along those lines, could not we write the genotype as a file as well? right now we wrote all of that in code so we have something like:
information - > code_encoding_the_information - > loading to metadata
instead of:
information_in_the_file -> loading_to_metadata. 

Just curious about your thoughts in this one.

Reply:
For the subject-specific genotype I think it makes more sense to code it; the `metadata.yml` file can really only correspond to a single session/subject at a time, which is why we're using it here for storing all global metadata and merging it (via the `deep_dict_update`) with custom metadata per session.